packaging_automation/common_tool_methods.py

```diff
@@ -507,3 +514,14 @@ def create_pr(gh_token: str, pr_branch: str, pr_title: str, repo_owner: str, pro
 
 def create_pr_with_repo(repo: Repository, pr_branch: str, pr_title: str, base_branch: str):
     return repo.create_pull(title=pr_title, base=base_branch, head=pr_branch, body="")
+
+
+def stat_get_request(request_address: str, request_type: RequestType, session):
+    request_log = RequestLog(request_time=datetime.now(), request_type=request_type.name)
+    session.add(request_log)
+    session.commit()
+    result = requests.get(request_address)
+    request_log.status_code = result.status_code
+    request_log.response = result.content.decode("ascii")
+    session.commit()
```

Comment:
Why there are two `session.commit()`s in this function?

Reply:
I add a log before making a get request to make sure that we can see a log even if server does not respond. I add a try except logic to write detailed log about the error